keyboards/sck/gtm/keymaps/default/keymap.c

```diff
@@ -0,0 +1,28 @@
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+LAYOUT(
+    KC_1, KC_2, KC_3, KC_4, KC_5,
+    KC_6, KC_7, KC_8, KC_9, KC_0, KC_ENTER),
+
+};
+
+void matrix_init_user(void) {
+  debug_config.matrix = 1;
+  debug_config.keyboard = 1;
+  debug_config.enable = 1;
+}
+
+
+
+void encoder_update_user(int8_t index, bool clockwise) {
+  if (index == 0) {
```

Comment:
If you only have the one, you don't really need to have the index.  

However, there is also no harm in having it, either.

Reply:
so then lines 19-27 can be removed?

Reply:
Just the enclosing `if (index == 0) { ... }`.

Reply:
ok I think I understand. that should be correct now. if not please let me know.